man/ocamlc.m

```diff
@@ -986,7 +986,7 @@ compilation in any way (even if it is fatal).  If a warning is enabled,
 
 .IP
 The default setting is
-.BR \-w\ +a\-4\-6\-7\-9\-27\-29\-32..39\-41\-42\-44\-45\-48\-50\-60 .
+.BR \-w\ +a\-4\-6\-7\-9\-27\-29\-32..39\-40\-41\-42\-44\-45\-48\-50\-60 .
```

Comment:
You should merge this into ...`\-29\-32..42\-44`... instead of this confusing sequence of an interval followed by single points.

Reply:
Fixed